doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -14,38 +14,58 @@
 
 <a name="2.4"></a>
 ## 2.4 Profiles Reference 
-
-Define the profiles used, why they're important, and what is being certified..
+Different vendors have different types of VNFs to serve different use-cases. A VNF like Broadband Network Gateway (BNG) would require high netorking throughout whereas a VNF like Mobility Management Entity (MME) woud require high computing performance. As such, BNG would require high KPI values for network throughput and MME would require high CPU performance KPIs like Index Score, Instructions Per Second (IPS) etc. The target NFVI to cater these needs woud have different characteristics. Depending on VNF's requirements, the NFVI can be categorised into below profiles:
+* Basic (B) profile for standard computing
+* Compute intensive (C) profile where predictable computing performance is expected and
+* Network intensive (N) profile offerring low latency and high networking throughout
+Similarly, different NFVI vendors may specialise in different hardware profiles and some may specialise in both VNFs and NFVI. To cater to different needs from multiple NFVI vendors, CNTT allows different types of NFVI certification based on their types of [profile](../../../ref_model/chapters/chapter02.md#2.3)
+ * Certify Vendor NFVI Hardware solution: This allows for certification of only NFVI.
+ * Certify Vendor NFVI Hardware and Software Solution: This allows for certification for NFVI running a particular VNF.
 
 <a name="2.5"></a>
 ## 2.5 Compliance, Verification, and Certification
-
-Define the framework and process for NFVI certification..
+The below set of steps define the compliance, verification and certification process for NFVI
+* Based on VNF's requirements, NFVI profile is selected - B, C, N
+* NFVI readiness is checked for certification.
+* The test VNFs are on-boarded using automation scripts on the NFVI.
+* VNF on-boarding is validated by running functional tests to ensure that the on-boarding is successful.
+* VNF performance tests are executed and NFVI KPIs are recorded during the tests.
+* KPI comparison is run to compare NFVI KPIs with Golden KPIs, which serve as a reference for NFVI certification.
+* If NFVI KPIs meet Golden KPIs, NFVI is certified and granted a certification badge.
+* If NFVI KPIs do not meet Golden KPIs, no certification is provided.
 
 <a name="2.6"></a>
 ## 2.6 Entry & Exit Criteria
+**Entry criteria**: Before entering into NFVI certification, NFVI needs to satisfy the following requirements as entry pass:
+* Lab Requirements*: The NFVI lab needs to be setup according to RA1/RA2 as defined by CNTT and should be in the required state.
 
```

Comment:
Add more Entrance Criteria:

**OPNFV Entrance Criteria includes _Demonstration of Reference Architecture Implementation and Delivery of the following:_**

1. **Design & Requirements**
  - Design, Configuration, Features, SLAs, and Capability documentation complete
  - Users stories / Adherence to CNTT Model principles and guidelines
  - Chosen Reference Architecture Matches a Reference Architecture from the product catalog
2. **Environment**
  - Lab assets/resources and respective s/w rev levels are specified, with confirmation of compatibility across external systems
  - Tenant needs identified
  - All connectivity, network, image, VMs, delivered with successful pairwise tests
  - Lab instrumented for proper monitoring
3. **Planning & Delivery**
  - Kickoff / Acceptance Criteria reviews performed
  - Delivery commitments, timelines, and cadence accepted
  - Confirm backward compat. across software/flavour rev levels
4. **Data / VNFs / Security**
  - Images, Heat Templates, Preload Sheets available
  - Images uploaded to tenant space
  - External system test data needs identified
  - Owners (NFVI, VNF, PTL, etc) documented
  - Security Compliance Satisfied (Refer to CNTT Chapter XXXX Security for additional tests, scans, and vulnerabilities validations)
5. **Test Case Contributions**
  - VNF Developer/Supplier validations to be performed documented and supplied
  - NFVi validations to be performed supplied (e.g. echo, manifest)
  - Test to ensure users are added and have correct privileges for the tenant
  - Test to ensure quota against submitted request for the respective tenant
  - Test to ensure custom flavours against submitted request for respective tenants

Reply:
Added

Reply:
Let me know if we can resolve this.

Reply:
Looks good.  Closing/Resvolving now.